guides/source/autoloading_and_reloading_constants.md

```diff
@@ -415,11 +415,11 @@ module Admin
 end
 ```
 
-In `zeitwerk` mode that does not matter anymore, you can pick either style.
+In `zeitwerk` mode that does not matter anymore. You can pick either style.
 
-The resolution of a constant could depend on load order, the definition of a class or module object could depend on load order, there was edge cases with singleton classes, oftentimes you had to use `require_dependency` as a workaround, .... The guide for `classic` mode documents [these issues](autoloading_and_reloading_constants_classic_mode.html#common-gotchas).
+In `classic` mode, the resolution of a constant could depend on load order, the definition of a class or module object could depend on load order, there were edge cases with singleton classes, oftentimes you had to use `require_dependency` as a workaround, .... The list goes on. The guide for `classic` mode documents [these issues](autoloading_and_reloading_constants_classic_mode.html#common-gotchas).
 
-All these problems are solved in `zeitwerk` mode, it just works as expected, and `require_dependency` should not be used anymore, it is no longer needed.
+All these problems are solved in `zeitwerk` mode. It just works as expected and `require_dependency` should not be used anymore. It is no longer needed.
```

Comment:
```suggestion
All these problems are solved in `zeitwerk` mode. It just works as expected. `require_dependency` should not be used anymore, because it is no longer needed.
```